openlibrary/catalog/edition_merge/templates/index.html

```diff
@@ -32,7 +32,7 @@ <h1>Merge duplicate editions</h1>
 
 {{ pager() }}
 
-{{ total }} items need to be merged ({{ easy }} easy)<p>
+{{ total }} items need to be merged ({{ easy }} easy)</p>
```

Comment:
this doesn't look right to me. I think the problem here is that the macro pager creates a p tag but doesn't xlose it. Why don't we change the pager method to accept a string

e.g. `{{ pager( '{{ %s }} items need to be merged ({{ %s }} easy)'%(total, easy) }}`

OR let's not open that p tag in the macro pager.

Reply:
okay working on it